tests/spread/plugins/craft-parts/matter/snapcraft.yaml

```diff
@@ -0,0 +1,59 @@
+name: matter-lighting
+summary: Matter plugin test
+description: An lighting application to test the matter plugin.
+version: "1.0"
+
+base: core22
+
+grade: stable
+build-base: core22
+confinement: strict
+
+layout:
+  /mnt:
+    bind: $SNAP_COMMON/mnt
+
+apps:
+  matter-lighting:
+    daemon: simple
+    command: bin/lighting-app
+    install-mode: disable
+    plugs:
+      - network
+      - network-bind
+      - bluez
+      - avahi-control
+
+parts:
+  matter-sdk:
+    plugin: matter
+    matter-sdk-version: "1536ca20c5917578ca40ce509400e97b52751788" # use this commit with ptpython version fix; needs to be updated once matter sdk have a stable release
+    matter-zap-version: "v2023.11.13"
+    override-build: |
+      craftctl default
+
+      # Remove snapcraft part related environment variables
+      cd matter
+      sed -i '/^CRAFT_PART_/d' matter_sdk_env
+      sed -i '/^SNAPCRAFT_PART_/d' matter_sdk_env
+
+  lighting:
+    plugin: nil
+    after: [matter-sdk]
+    override-build: |
+      craftctl default
```

Comment:
I believe there are defaults for the nil plugin.

Reply:
Removed.